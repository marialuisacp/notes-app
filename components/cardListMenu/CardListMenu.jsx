import React from 'react';
import StyledCardListMenu from './styles/CardListMenu.styles';

const CardListMenu = () => (
  <StyledCardListMenu></StyledCardListMenu>
);

export default CardListMenu;