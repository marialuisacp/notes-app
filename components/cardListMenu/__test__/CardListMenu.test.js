import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import CardListMenu from '../CardListMenu';

test('should render component CardListMenu', () => {
  const { fragment } = render(<CardListMenu></CardListMenu>);
  expect(fragment).toMatchSnapshot();
});
