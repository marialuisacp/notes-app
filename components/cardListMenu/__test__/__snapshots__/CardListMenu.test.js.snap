// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component CardListMenu 1`] = `undefined`;
